### DISCUSSION TITLE
"cloudflare images" confusion, possible support?

### DISCUSSION THREAD

**Discussion by LukeClancy** [frostmark/carrierwave-cloudflare#12] [Ideas] [answered]
Cloudflare recently created "cloudflare images", which is notably different from "cloudflare image resizing". Given that cloudflare did not store things in the cloud before "cloudflare images", the name of this gem ([carrierwave-cloudflare](https://github.com/resume-io/carrierwave-cloudflare)) makes sense at that moment in time. However now that cloudflare does store things in the cloud, it is a little confusing (_how many times have I said cloud? eugh_).

Are there plans to support "cloudflare images"? If not, I would appreciate it if the README summary explicitly noted that "cloudflare images" are not supported, and that this gem does not store files in the cloud.

Thank you for your time,

Luke

**Comment by frostmark** [comment, marked as answer]
Yeah this gem does not support image uploading to cloudflare (and I don't have plans to add it), just transforming on the fly. Added comment about that to readme. Thanks!
